packages-exp/auth-exp/src/core/user/token_manager.ts

```diff
@@ -147,21 +158,33 @@ export class StsTokenManager {
       );
       manager.expirationTime = expirationTime;
     }
+    if (isPassthroughMode) {
+      _assert(
+        typeof isPassthroughMode === 'boolean',
+        AuthErrorCode.INTERNAL_ERROR,
+        {
+          appName
+        }
+      );
+      manager.isPassthroughMode = isPassthroughMode;
+    }
     return manager;
   }
 
   toJSON(): object {
```

Comment:
tests for to/from JSON?

Reply:
Good catch, added / updated tests in token_manager.test.ts